services/tmc/next.config.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable i18next/no-literal-string */
+/* eslint-disable @typescript-eslint/no-var-requires */
+const externallyEmbeddableIFrameResponseHeaders =
+  require("./src/shared-module/utils/responseHeaders").externallyEmbeddableIFrameResponseHeaders
+const svgoConfig = require("./src/shared-module/utils/svgoConfig")
+
+const config = {
+  eslint: {
+    ignoreDuringBuilds: true,
+  },
+  async headers() {
+    return [
+      {
+        source: "/(.*)",
+        headers: externallyEmbeddableIFrameResponseHeaders,
+      },
+      // This application is meant to be used with a sandboxed iframe.
+      // That causes that we need cors headers for fonts.
+      {
+        source: "/(.*).woff2",
+        headers: [{ key: "Access-Control-Allow-Origin", value: "*" }],
```

Comment:
This can be removed now since this header is added to all requests in `externallyEmbeddableIFrameResponseHeaders`